web/src/layout/StackedLayout.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-    <Navbar v-if="!loading"></Navbar>
+    <!-- <Navbar v-if="!loading"></Navbar> -->
     <div v-if="!loading">
+        <Navbar @profileSettingHandler="profileToggle"></Navbar>
         <router-view v-slot="{ Component }">
             <transition name="slide-fade" mode="out-in">
                 <component :key="$route.name" :is="Component" />
             </transition>
         </router-view>
-    </div>
-    <div class="text-red-500" v-if="errors && !loading">
-        <p v-for="(error, i) in errors">
-            {{ error.title }}
-            <span @click="removeError(i)" class="cursor-pointer">X</span>
-        </p>
+        <SettingContent :show="profileSettingIsShow" @closeDialog="profileToggle" />
```

Comment:
If you figured out how to center it globally, I guess moving this back into where it's used is better. So in Navbar for example.